spec/models/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+# Title must not be blank.
+# Title must not exceed 250 characters.
+# CommentsCounter must be an integer greater than or equal to zero.
+# LikesCounter must be an integer greater than or equal to zero.
+
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  # tests go here
+  user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  subject { Post.new(user:, title: 'Hello', text: 'This is my first post') }
+
+  before { subject.save }
+
+  context 'Test post validations' do
+    it 'is valid with valid attributes' do
+      expect(subject).to be_valid
+    end
+
+    it 'title must not be blank' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'Title must not exceed 250 characters' do
+      long_title = 'long ' * 55
+      subject.title = long_title
+      expect(subject).to_not be_valid
+    end
```

Comment:
* Kindly Add unit specs files for all of your models' methods and validations. according to the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)  ( ie Has unit tests for `User`, `Post`, `Comment`, and `Like` model )

Reply:
Sure! An apologize, I completely forgot about it 😞